openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -3,8 +3,35 @@
 
 Includes namespacing, caching, and course overrides for waffle flags.
 
-For testing WaffleFlags, see testutils.py.
-For testing WaffleSwitchNamespace, use its context manager.
+Usage:
+
+For Waffle Flags, first set up the namespace, and then create flags using the
+namespace.  For example:
+
+    WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(namespace='course_experience', log_prefix=u'Course Experience: ')
+
+    HIDE_SEARCH_FLAG = WaffleFlag(WAFFLE_FLAG_NAMESPACE, 'hide_search')
+   UNIFIED_COURSE_TABS_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'unified_course_tabs')
+ 
+You can check these flags in code using the following:
+
+    HIDE_SEARCH_FLAG.is_enabled()
+    UNIFIED_COURSE_TABS_FLAG.is_enabled(course_key)
+
+To test these WaffleFlags, see testutils.py.
+
+    WAFFLE_SWITCHES = WaffleSwitchNamespace(namespace=WAFFLE_NAMESPACE, log_prefix=u'Grades: ')
```

Comment:
I think you need an introductory sentence to mention that you are now talking about switches.